includes/class.llms.course.data.php

```diff
@@ -164,6 +164,14 @@ public function get_lesson_completions( $period = 'current' ) {
 
 		$lessons = implode( ',', $this->post->get_lessons( 'ids' ) );
 		// phpcs:disable WordPress.DB.PreparedSQL.InterpolatedNotPrepared
+
+		// It is possible to create a course but not have any lessons yet. Return zero now
+		// instead of causing a MySQL error by leaving "post_id IN ( {$lessons} )"
+		// blank in the query.
```

Comment:
We don't have an official CS rule about this but generally we rwrite multiline comments using `/** */` style comments. However, I don't think it's necessary to describe the bug in a comment regardless.
```suggestion
		// Return early for courses without any lessons.
```

Reply:
Ah yes, that makes sense.  I committed your suggested change to the comment.